build/lib/compendium-pack.ts

```diff
@@ -331,6 +331,12 @@ class CompendiumPack {
         const convertOptions = { to: to === "ids" ? "id" : "name", map } as const;
 
         // Convert UUIDs found in places particular to certain item types
```

Comment:
Maybe the new stuff should have been put above this comment?

Reply:
Switched to use itemIsOfType() instead.

Reply:
Not actually resolved. The thing I'm concerned about is the proximity of the comment with what its commenting about. The way you have it, it looks like suppressedFeatures is the only thing this comment is for, but the if/else below is also relevant.

But I'm thinking more about it and we can just remove the comment entirely.